py-polars/polars/functions/whenthen.py

```diff
@@ -66,7 +66,7 @@ def when(
     │ 4   ┆ 0   ┆ 1   │
     └─────┴─────┴─────┘
 
-    Or with multiple `when, thens` chained:
+    Or with multiple `when, then`s chained:
```

Comment:
I think this syntax breaks Sphinx.

```suggestion
    Or with multiple when-then operations chained:
```

Reply:
I figured that was it once I saw that build break. Sorry about that, will fix up now.